SimG4CMS/Calo/test/python/runWithGun_cfg.py

```diff
@@ -182,20 +185,20 @@
     EkinParticles           = cms.vstring(),
     Verbosity               = cms.untracked.int32(2)
 )
-process.g4SimHits.Watchers = cms.VPSet(cms.PSet(
-    CheckForHighEtPhotons = cms.untracked.bool(False),
-    TrackMin     = cms.untracked.int32(0),
-    TrackMax     = cms.untracked.int32(0),
-    TrackStep    = cms.untracked.int32(1),
-    EventMin     = cms.untracked.int32(0),
-    EventMax     = cms.untracked.int32(0),
-    EventStep    = cms.untracked.int32(1),
-    PDGids       = cms.untracked.vint32(),
-    VerboseLevel = cms.untracked.int32(0),
-    G4Verbose    = cms.untracked.bool(True),
-    DEBUG        = cms.untracked.bool(False),
-    type      = cms.string('TrackingVerboseAction')
-))
+#process.g4SimHits.Watchers = cms.VPSet(cms.PSet(
```

Comment:
That watcher is not working right now. I shall work on it later - make it work and then uncomment these lines